docs/non-normative/how-to-write-conventions/status-metrics.md

```diff
@@ -0,0 +1,87 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Status Metrics
+aliases: [status-metrics]
+--->
+
+# State Metrics
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [Definition](#definition)
+- [Design](#design)
+  - [Instrument](#instrument)
+  - [Should it be an Entity Attribute?](#should-it-be-an-entity-attribute)
+  - [Naming](#naming)
+
+<!-- tocstop -->
+
+## Definition
+
+For the purposes of this document, a "status metric" is a metric used to
+represent something being in a particular state at a given time from a closed
+set of distinct possible state values.
+
+## Design
+
+To represent a metric of this nature, the design will look like the following:
+
+```yaml
+id: metric.example.status
+type: metric
+metric_name: example.status
+stability: development
+brief: "The current state."
+note: |
+  A timeseries is produced for every possible value of `example.state`. The
+  value of this metric is 1 for a state if it is currently in said state, and
+  is 0 for all other states.
+instrument: updowncounter
+unit: "1"
+attributes:
+- ref: example.state
+```
+
+### Instrument
+
+The metric is
+[instrumented as an `UpDownCounter`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#consistent-updowncounter-timeseries)
+rather than a `Gauge`. This is a deliberate choice, as it is a reasonable use 
+case to count objects that are in a particular state. Since the metric value is
+either `0` or `1` for a given state attribute value, this means you can do a
+simple sum aggregation to count instances of particular states.
+
+### Should it be an Entity Attribute?
+
+It is possible to make the status a descriptive attribute on an entity (**not**
+an identifying attribute). The primary upside to making a state metric is for
+tracking state changes over time; as an entity attribute, the history of state
+changes would not necessarily be tracked.
+
+### Naming
+
+If there is particular verbiage for this sort of thing within the instrumentation
```

Comment:
```suggestion
if there is already well-known naming conventions within the instrumentation
```
Maybe? The doc is very well written but "for this sort of thing" caught me off guard here :D